docs/DeveloperGuide.md

```diff
@@ -270,15 +270,24 @@ _{Explain here how the data archiving feature will be implemented}_
-
+| Priority | As a …​                                 | I want to …​                               | So that I can…​                                                     |
+| -------- | ------------------------------------------ | --------------------------------------------- | ---------------------------------------------------------------------- |
+| `* * *`  | tutor for the first use                    | see all commands available                    | recall commands and use them properly when I forget how to use the app |
+| `* * *`  | tutor                                      | add a new student or professor                |                                                                        |
+| `* * *`  | tutor                                      | delete a student or professor                 | remove entries that I no longer need                                   |
+| `* * *`  | tutor                                      | find a person by name or module               | locate details of persons without having to go through the entire list |
+| `* * *`  | tutor                                      | assign tasks to students                      |                                                                        |
+| `* * *`  | tutor                                      | delete tasks assigned before                  | remove the outdated assignments for students                           |
+| `* * *`  | tutor                                      | mark students' tasks as done                  | record students' progress more easily                                  |
+| `* *`    | tutor teaching online                      | access the web links used for teaching        | access information from teaching websites immediately                  |
+| `* *`    | tutor for several modules                  | organize student data according to module     | manage my tasks of different modules in an organised manner            |
+| `* *`    | student and tutor                          | organise my tasks in order of deadline        | manage my time better                                                  |
+| `*`      | easily frustrated tutor                    | search up contacts on the search bar fuss-free| save time used for fighting the app over syntax issues                 |
+| `*`      | busy tutor                                 | list certain people I interacted frequently   | save time searching their name whenever I start app                    |
+| `*`      | tutor with many persons in the contact book| sort persons by name                          | locate a person easily                                                 |
+| `*`      | responsible tutor                          | track students' progress on their assignments | identify and reach out to those who need help                          |
```

Comment:
Same as the previous comment!

Reply:
i think the line of 286 should be " ** ", since it's not linked to the main function of our app directly.